taf/updater/updater.py

```diff
@@ -384,6 +426,9 @@ def _update_named_repository(
             commits=commits,
         )
 
+        # read dependencies.json and set them to the parent authentication repository
+        auth_repo.set_dependencies(_load_dependencies_json(auth_repo))
```

Comment:
I would move this code to `repositoriesdb`. Actually, we already load `dependencies.json` there and can just set the attribute. All code which deals with instantiation of repositories should be in `repositoriesdb` to stay consistent with the current design

Reply:
does apparoch in [this](https://github.com/openlawlibrary/taf/pull/206/commits/d84d01daf32cec984ca27cd87df6b6c546d41151) commit make sense?